### PR TITLE
Geometric Multiscale Documentation

### PR DESCRIPTION
**Pull request by ezonta**
### Content of PR
Added a short section on geometric multiscale mapping and how it is configured that can be extended further upon expansion of preCICE's geometric multiscale capabilities (e.g., radial geometric multiscale coupling, tutorials on 1D-3D coupling, etc.)

### CODE REVIEW

**Comment by ezonta** [comment]
@MakisH I just learned that I have to add you as a collaborator to the repo before I can add you as a reviewer, so I will add you as a reviewer as soon as you accept the collaborator invite.

**Comment by uekerman** [comment]
> I just learned that I have to add you as a collaborator to the repo before I can add you as a reviewer, so I will add you as a reviewer as soon as you accept the collaborator invite.

I guess it's the other way around. 
@ezonta you have to become a collaborator in the upstream repo. I just added you.

**Comment by ezonta** [comment]
> > I just learned that I have to add you as a collaborator to the repo before I can add you as a reviewer, so I will add you as a reviewer as soon as you accept the collaborator invite.
> 
> I guess it's the other way around. @ezonta you have to become a collaborator in the upstream repo. I just added you.

Oh perfect, thank you!

File: pages/docs/configuration/configuration-mapping.md
Comment (MakisH) [inline pages/docs/configuration/configuration-mapping.md]: I think that this definition is a bit confusing. In 1D-3D, the main difference is whether we are coupling to one or to multiple points of the 1D mesh. Check again the OFW14 paper: https://mediatum.ub.tum.de/node?id=1515271&change_language=en

We should probably refer to this paper somewhere here anyway.

File: pages/docs/configuration/configuration-mapping.md
Comment (MakisH) [inline pages/docs/configuration/configuration-mapping.md]: Move the image here (e.g., to https://github.com/precice/precice.github.io/tree/master/images/docs/configuration) and name it accordingly (e.g., `doc-mapping-geometric-multiscale.png`). Also provide the source file of the image in the same folder.

Note that the `alt=` is only mentioning axial mapping, while the picture also shows radial.

File: pages/docs/configuration/configuration-mapping.md
Comment (MakisH) [inline pages/docs/configuration/configuration-mapping.md]:
```suggestion
The `type,` which can be either `"spread"` or `"collect"`, refers to whether the participant spreads data from one mesh node to multiple nodes or collects data from multiple mesh nodes into one node. In this case `MyMesh1` would be the higher dimensional one, as data needs to be spread to multiple nodes in order for `MyMesh2` to read it.
```

**Review by MakisH — COMMENTED** [review]
Thank you for contributing! A few comments to start the discussion.

I also converted this to a draft, let's decide based on our strategy with https://github.com/precice/precice/pull/1439

File: pages/docs/configuration/configuration-mapping.md
Comment (MakisH) [reply pages/docs/configuration/configuration-mapping.md]: I moved the images into the repository. They don't render in the preview here, but I have checked locally.

**Review by MakisH — COMMENTED** [review]
Thanks for the updates! To save some iterations, I went ahead and directly edited the text. It looks good for now, but we can only merge when the feature is merged to develop.

**Comment by MakisH** [comment]
I changed the target branch to `precice-v3`, which is meant to merge once v3 is out.
So, @ezonta, if you feel that this is ready (still marked as draft), we could already merge it and move on to the next step, focusing on the cases.

**Review by MakisH — APPROVED** [review]

**Comment by MakisH** [comment]
@ezonta looking at the [files changed](https://github.com/precice/precice.github.io/pull/205/files), I see a few unreated changes. Could you maybe rebase on the latest state of the `precice-v3` branch and force-push?

**Comment by ezonta** [comment]
> @ezonta looking at the [files changed](https://github.com/precice/precice.github.io/pull/205/files), I see a few unreated changes. Could you maybe rebase on the latest state of the `precice-v3` branch and force-push?

I can do that! Do you know if there is way to rebase from GitHub web? I haven't yet cloned the repo, because these were only text changes. But I can also just clone the repo real quick to do it.

**Comment by MakisH** [comment]
> > @ezonta looking at the [files changed](https://github.com/precice/precice.github.io/pull/205/files), I see a few unreated changes. Could you maybe rebase on the latest state of the `precice-v3` branch and force-push?
> 
> I can do that! Do you know if there is way to rebase from GitHub web? I haven't yet cloned the repo, because these were only text changes. But I can also just clone the repo real quick to do it.

I don't think one can do that directly via the UI... :-/

**Comment by ezonta** [comment]
> > > @ezonta looking at the [files changed](https://github.com/precice/precice.github.io/pull/205/files), I see a few unreated changes. Could you maybe rebase on the latest state of the `precice-v3` branch and force-push?
> > 
> > 
> > I can do that! Do you know if there is way to rebase from GitHub web? I haven't yet cloned the repo, because these were only text changes. But I can also just clone the repo real quick to do it.
> 
> I don't think one can do that directly via the UI... :-/

I cloned the repo and force pushed a rebased version, hope it worked out.

**Comment by MakisH** [comment]
> I cloned the repo and force pushed a rebased version, hope it worked out.

Strange... Looking at https://github.com/precice/precice.github.io/pull/205/files, I still see unrelated changes (and unrelated commits).

**Comment by MakisH** [comment]
I rebased on `precice-v3`:

1. `git checkout precice-v3 && git pull`
2. `gh pr checkout 205`
3. `git rebase precice-v3`
4. `git push --force`

I now only see the relevant files.

**Review by MakisH — APPROVED** [review]
This is how this section looks like now:

Screenshot from 2023-08-08 16-15-37

Unless we change anything else in https://github.com/precice/precice/pull/1675, this is ready to merge once the PR in preCICE is merged.

**Review by MakisH — APPROVED** [review]

**Pull request merged**
